messaging/src/main/java/ids/messaging/protocol/http/IdsHttpService.java

```diff
@@ -416,16 +437,16 @@ private Response sendRequest(final Request request,
     }
 
     /**
-     * Logs the request message to be send in debug-level log.
+     * Logs the request message to be send in log.
      *
      * @param request The request message to be send.
      * @throws IOException If Buffer could not be accessed.
      */
-    private void debugLogRequest(final Request request) throws IOException {
+    private void logRequest(final Request request) throws IOException {
         final var requestCopy = request.newBuilder().build();
         final var buffer = new Buffer();
         requestCopy.body().writeTo(buffer);
-        log.debug("Sending request message: [code=(IMSMED0135), request=({})]", buffer.readUtf8());
+        log.info("Sending request message: {}", buffer.readUtf8());
```

Comment:
Shall we not be sticking to the 'code' to have an easy chance to find the line, where a log was triggered?

Reply:
I opened an Issue https://github.com/International-Data-Spaces-Association/IDS-Messaging-Services/issues/536 so it can be introduced on its own PR.